src/slack/vacuumlabs/constants.js

```diff
@@ -1,4 +1,4 @@
-export const SLACK_URL = 'https://vacuumlabs.slack.com'
+export const SLACK_URL = "https://vacuumlabs.slack.com";
```

Comment:
you probably have some formatter turned on. Prettier is not configured for this repo yet, so you may need to turn it off for now. and instead, please use ESLint's autofix feature to fix some errors in other changed files as well.

the next steps:
- revert the changes in this file and reapply your logic changes
- run `yarn eslint --fix .` (and ignore the tons of listed errors)

commit these changes separately or squash them into your last commit and force push, your choice (it is not many changes, easily reviewable again).